challenges/cat-photo-gallery/script.js

```diff
@@ -0,0 +1,36 @@
+window.onload = getRandomCatPicture ()
+
+async function getRandomCatPicture () {
+    await fetch("https://cataas.com/cat?json=true")
+    .then(response => response.json())
+    .then(randomCat => {
+        const image = document.querySelector("img");
+        return image.src = "https://cataas.com" + randomCat.url;
+    }).catch(error => "Error: " + error)
+}
```

Comment:
in case if there an error has occurred, what would `catch(error => "Error: " + error)` do? 
there no `return` statement in it, nor `console.error()`...